src/fileAttachment.mjs

```diff
@@ -10,11 +10,38 @@ async function remote_fetch(file) {
   return response;
 }
 
-async function dsv(file, delimiter, {array = false, typed = false} = {}) {
+function asTyper(type) {
+  if (typeof type === "function") {
+    return type;
+  }
+  switch (`${type}`) {
+    case "number": return Number;
+    case "string": return String;
```

Comment:
in autoType the string gets trimmed. probably not something to include in casting, but will be different